instance/tests/api/test_instance.py

```diff
@@ -114,22 +115,27 @@ def test_provision_github_branch_deleted(self, mock_get_commit_id_from_ref):
     @patch('instance.api.instance.provision_instance')
     def test_provision(self, mock_provision_instance, mock_get_commit_id_from_ref):
         """
-        POST /:id/provision
+        POST /:id/provision - Instance is in a state that allows provisioning
         """
+        instance_factories = (
+            SingleVMOpenEdXInstanceFactory, RunningSingleVMOpenEdXInstanceFactory,
+            ErrorSingleVMOpenEdXInstanceFactory, ConfigFailedSingleVMOpenEdXInstanceFactory,
+            TerminatedSingleVMOpenEdXInstanceFactory
```

Comment:
Nit: ddt would be better, and this could be a list of status values passed to `SingleVMOpenEdXInstanceFactory` if you wanted.